webapp/packages/plugin-navigation-tree/src/TreeNew/DataTransformers/rootTransformers.ts

```diff
@@ -10,7 +10,7 @@ import type { INodeState } from '../INodeState';
 import type { TreeDataTransformer } from './TreeDataTransformer';
 
 export function rootNodeStateTransformer(root: string): TreeDataTransformer<INodeState> {
-  return function rootNodeStateTransformer(nodeId, data) {
+  return function rootNodeStateTransformer(treeData, nodeId, data) {
```

Comment:
not in use here and below also. is this for future use or just forgot to delete it?

Reply:
i personally prefer to left full name than _